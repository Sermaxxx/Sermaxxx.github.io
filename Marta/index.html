<!DOCTYPE html>
<html lang="en">

<head>
  <meta charset="UTF-8">
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  <meta http-equiv="X-UA-Compatible" content="ie=edge">
  <title>Document</title>
  <!-- Stylesheet -->
  <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@400;600;700&display=swap" rel="stylesheet">
  <link rel="stylesheet" href="css/style.min.css">
</head>

<body>

  <header id="header" class="header" style="background-image:url(../img/header-bg.jpg)">
    <div class="container">
      <div class="header__inner">
        <div class="header-top">
          <div class="header__logo">
            <a href="#"><img src="img/logo.png" alt=""></a>
          </div>
          <div class="header__nav">
            <nav>
              <ul>
                <div class="close-menu">X</div>
                <li><a href="#header">Home</a></li>
                <li><a href="#features">Work</a></li>
                <li><a href="#costumer">About</a></li>
                <li><a href="#contact">Contact</a></li>
              </ul>
            </nav>
            <div class="opacityyy"></div>
          </div>
          <div class="header__log">
            <div class="header__log-login">
              <a href="">Login</a>
            </div>
            <div class="header__log-signup">
              <a href="">Sign Up</a>
            </div>
          </div>
        </div>
        <div class="header-main">
          <div class="header__title">
            We Build Apps & Websites
            for Your Business
          </div>
          <div class="header__text">
            For startups, For businesses & For you
          </div>
          <div class="header__btn">
            <button>Request a quote</button>
          </div>
        </div>
      </div>
      <div class="header__menu-btn">
        <div></div>
        <div></div>
        <div></div>
      </div>
    </div>
  </header>

   <!-- Features  -->
   <section class="features" id="features">
    <div class="container">
      <div class="features__inner">
        <div class="features__title">
          We are specialised in
        </div>
        <div class="features__items">
          <div class="features__item">
            <div class="features__item-img">
              <img src="img/features-1.svg" alt="">
            </div>
            <div class="features__item-title">
              UI/UX Design
            </div>
            <div class="features__item-text">
              In hac habitasse platea dictumstmus adipiscing fermentum quam volutpat aliquam. Integer cilisis tristique.
            </div>
          </div>
          <div class="features__item">
            <div class="features__item-img">
              <img src="img/features-2.svg" alt="">
            </div>
            <div class="features__item-title">
              UI/UX Design
            </div>
            <div class="features__item-text">
              In hac habitasse platea dictumstmus adipiscing fermentum quam volutpat aliquam. Integer cilisis tristique.
            </div>
          </div>
          <div class="features__item">
            <div class="features__item-img">
              <img src="img/features-3.svg" alt="">
            </div>
            <div class="features__item-title">
              UI/UX Design
            </div>
            <div class="features__item-text">
              In hac habitasse platea dictumstmus adipiscing fermentum quam volutpat aliquam. Integer cilisis tristique.
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>

   <!-- About  -->
  <section class="about" id="about">
    <div class="container">
      <div class="about__inner">
        <div class="about___items">
          <div class="about__item">
            <div class="about__item-texts">
              <div class="about__item-title">
                Experienced Designers &
                Developers
              </div>
              <img class="about__item-text-bg" src="img/about-text-bg.svg" alt="">
              <div class="about__item-text">
                Nam dapibus nisl vitae elit fringilla rutrum. Aenean sollicitudin, erat a elementum rutrum, neque sem
                pretium metus, quis mollis nisl nunc et massa.
              </div>
              <div class="about__item-text">
                Vestibulum sed metus in lorem tristique ullamcorper id vitae erat.
              </div>
            </div>
            <div class="about__item-img">
              <img src="img/about-1.svg" alt="">
            </div>
          </div>
          <div class="about__item">
            <div class="about__item-img">
              <img src="img/about-2.svg" alt="">
            </div>
            <div class="about__item-texts">
              <div class="about__item-title">
                Experienced Designers &
                Developers
              </div>
              <img class="about__item-text-bg2" src="img/about-text-bg2.svg" alt="">
              <div class="about__item-text">
                Nam dapibus nisl vitae elit fringilla rutrum. Aenean sollicitudin, erat a elementum rutrum, neque sem
                pretium metus, quis mollis nisl nunc et massa.
              </div>
              <div class="about__item-text">
                Vestibulum sed metus in lorem tristique ullamcorper id vitae erat. Nulla mollis sapien sollicitudin
                lacinia lacinia. Nam dapibus nisl vitae elit fringilla rutrum. Aenean sollicitudin, erat a elementum
                rutrum, neque sem pretium metus, quis mollis nisl nunc et massa </div>
            </div>
          </div>
          <div class="about__item">
            <div class="about__item-texts">
              <div class="about__item-title">
                Experienced Designers &
                Developers
              </div>
              <img class="about__item-text-bg3" src="img/about-text-bg3.svg" alt="">
              <div class="about__item-text">
                Nam dapibus nisl vitae elit fringilla rutrum. Aenean sollicitudin, erat a elementum rutrum, neque sem
                pretium metus, quis mollis nisl nunc et massa. Nam dapibus nisl vitae elit fringilla rutrum. Aenean
                sollicitudin, erat a elementum rutrum, neque sem pretium metus, quis mollis nisl nunc et massa
              </div>
              <div class="about__item-text">
              </div>
            </div>
            <div class="about__item-img">
              <img src="img/about-3.svg" alt="">
            </div>
          </div>
        </div>
      </div>
    </div>
    </div>
  </section>

     <!-- Team   -->

  <section class="costumer" style="background-image:url(../img/costumer-bg.jpg)">
    <div class="container">
      <div class="costumer__inner">
        <div class="costumer__title">
          Don’t Just Take Our Words
        </div>
        <div class="costumer__items">
          <div class="costumer__item">
            <div class="costumer__item-worker">
              <div class="costumer__item-worker-img">
                <img src="img/worker-1.png" alt="">
              </div>
              <div class="costumer__item-worker-about">
                <div class="costumer__item-worker-name">
                  Arakat Bank
                </div>
                <div class="costumer__item-worker-place">
                  International Bank Ltd.
                </div>
              </div>
            </div>
            <div class="costumer__item-quotes">
              “
            </div>
            <div class="costumer__item-text">
              Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae
              ullamcorper metus. Sed sollicitudin ipsum.
            </div>
          </div>
          <div class="costumer__item" id="costumer">
            <div class="costumer__item-worker">
              <div class="costumer__item-worker-img">
                <img src="img/worker-2.png" alt="">
              </div>
              <div class="costumer__item-worker-about">
                <div class="costumer__item-worker-name">
                  Arakat Bank
                </div>
                <div class="costumer__item-worker-place">
                  International Bank Ltd.
                </div>
              </div>
            </div>
            <div class="costumer__item-quotes">
              “
            </div>
            <div class="costumer__item-text">
              Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae
              ullamcorper metus. Sed sollicitudin ipsum.
            </div>
          </div>
          <div class="costumer__item">
            <div class="costumer__item-worker">
              <div class="costumer__item-worker-img">
                <img src="img/worker-3.png" alt="">
              </div>
              <div class="costumer__item-worker-about">
                <div class="costumer__item-worker-name">
                  Arakat Bank
                </div>
                <div class="costumer__item-worker-place">
                  International Bank Ltd.
                </div>
              </div>
            </div>
            <div class="costumer__item-quotes">
              “
            </div>
            <div class="costumer__item-text">
              Fusce vehicula dolor arcu, sit amet blandit dolor mollis nec. Donec viverra eleifend lacus, vitae
              ullamcorper metus. Sed sollicitudin ipsum.
            </div>
          </div>
        </div>
      </div>
    </div>
  </section>

   <!-- Footer  -->

  <footer class="footer">
    <div class="container">
      <div class="footer__inner">
        <div class="footer__nav">
          <ul>
            <li><a href="">About Martaup</a></li>
            <li><a href="">Help</a></li>
            <li><a href="">Privacy Policy</a></li>
          </ul>
        </div>
        <div class="footer__copyright">
          © 2013-2018 All rights reserved.
        </div>
      </div>
    </div>
  </footer> 
<!--
 -->


  <!-- Scripts -->
  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
  <script src="js/libs.min.js"></script>
  <script src="js/main.js"></script>
</body>

</html>