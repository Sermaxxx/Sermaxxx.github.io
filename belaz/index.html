<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
    <link rel="stylesheet" href="bootstrap-4/css/bootstrap_min.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="fonts/fonts.css">
    <link rel="stylesheet" href="css/media.css">
    <script defer src="js/jquery.js"></script>
    <script defer src="js/script.js"></script>
</head>
<body>  
    <header> 
        <div class="container">
    <nav class="navbar navbar-expand-xl bg-dark">
        
            <a class="navbar-brand" href="#"><img src="img/logo.png" alt=""></a>
            <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent">
              <span class="navbar-toggler-icon"></span>
            </button>
          
            <div class="collapse navbar-collapse" id="navbarSupportedContent">
             </div> 
             <div class="work_time">
              <p><b>с 9:00 до 21:00</b> <br>
               без выходных</p>
              </div>
            <div class="p_number">
              <h3>+7 (818) 247-47-25</h3>
              <h3>+7 (818) 247-47-25</h3>
            </div>
            <div class="call_btn">
              <a href="#">Заказать звонок</a>
            </div>
          </nav>
        </header>
<!-- ============================================================================== -->
        <section class="welcome">
          <div class="container">
            <div class="block_txt block_padding">
             <h1>Оригинальные<br>
              запчасти <br>
              для Белаз</h1>
              <h3>с бесплатной доставкой <br>
                 в Ваш город!</h3>
                 <div>
                   <img class="block_ico" src="img/piggy-bank.png" alt="">
                   <p>Дилерские цены - ниже на 20%, <br>
                      работаем без посредников!</p>
                 </div>
                 <div>
                   <img class="block_ico" src="img/delivery-truck.png" alt=""> 
                   <p>Срочная бесплатная доставка <br>
                      за 7 дней</p>
                 </div>
                 <div>
                   <img class="block_ico" src="img/calendar.png" alt=""> 
                   <p>Даем рассрочку <br>
                      6 месяцев  </p>
                 </div>
                 <div>
                   <img class="block_ico" src="img/medal.png" alt=""> 
                   <p>Даем рассрочку <br>
                      6 месяцев  </p>
                 </div>

            </div>
          </div>
        </section>
<!-- =================================================================================== -->
        <section class="get_price block_padding">
          <div class="container">
            <div class="contact ">
            <h3>Получите прайс прямо сейчас <br>
              на электронную почту!</h3>
            <input type="email" placeholder="ВВЕДИТЕ ВАШ E-MAIL">
            <input type="text" placeholder="ВВЕДИТЕ ВАШ ТЕЛЕФОН">
           </div>
           <div class="get_5_min">
            <p>Прайс придет к Вам на почту<br> 
              в течение 5 минут!</p> 
            <img src="img/clock.png" alt="">
            <button class="get_price_btn">Получить Прайс</button>
          </div>
            </div>
        </section>
<!-- ============================================================================== -->
        <section class="about_details block_padding">
        <div class="container">
          <h2>Вам не придется ждать!</h2>
          <p>Любые запчасти для всех моделей Белаз <br>
              всегда в наличии!</p>
            <div class="about_ico row">
              <div class="col-lg-3 icons">
                <img src="img/engine.png" alt=""> 
                <p>Любые типы деталей <br>
                  от шайбы до двигателя!</p>
              </div>
              <div class="col-lg-3 icons"  >
                <img src="img/dump-truck.png" alt=""> 
                <p>Все модификации Белаз <br>
                    от 45т до 450 т</p>
              </div>
              <div class="col-lg-3 icons" >
                <img src="img/calendar_big.png" alt=""> 
                <p>Отгружаем в день заказа!</p>
              </div>
              <div class="col-lg-3 icons">
                <img src="img/locaition.png" alt=""> 
                <p>Доставим в любое место!</p>
              </div>
            </div>
           </div>
        </section>
<!-- ================================================================================ -->
        <section class="find_detail block_padding">
          <div class="container">
          <h3>Ищете конкретную деталь?</h3>
          <p>Если у Вас есть артикул - введите его в поле ниже и получите цену <br>
              ПРЯМО СЕЙЧАС!</p>
              <div class="f_detail">
                <input type="text" placeholder="ВВЕДИТЕ ВАШ E-MAIL">
                <input type="text" placeholder="ВВЕДИТЕ ВАШ ТЕЛЕФОН">
                <button>Узнать Цену</button>
              </div>
              <div class="call_us">
                <p>Мы перезвоним вам в течение 3 минут и назовем цену детали!</p>
                <p>Или Вы можете перезвонить нам сами:</p>
                <p> +7 (818) 247-47-25</p>
              </div>
            </div>
        </section>
<!-- ============================================================================= -->
        <section class="garantiya block_padding">
          <div class="container">
          <h2>Мы гарантируем сохранность деталей <br>
              во время перевозки!</h2>
              <div class="row">
                <div class="col-md-4">
                  <div><img src="img/garantiya.png" alt=""></div>
                  <p>Гарантия на целостность поставки <br>
                      прописана в договоре</p>
                      <a href="#">ПОДРОБНЕЕ</a>
                </div>
                <div class="col-md-4">
                  <div><img src="img/package.png" alt=""></div>
                  <p>Каждая запчасть доставляется с завода <br>
                      в жесткой упаковке</p>
                      <a href="#">ПОДРОБНЕЕ</a>
                </div>
                <div class="col-md-4">
                  <div><img src="img/quality_control.png" alt=""></div>
                  <p>Дополнительный контроль качества <br>
                      нашими специалистами</p>
                      <a href="#">ПОДРОБНЕЕ</a>
                </div>
              </div>
          </div>
        </section>
<!-- ========================================================================== -->
        <section class="sales block_padding">
          <div class="container">
            <h2>У нас разработаны бонусные системы <br>
                и скидки для наших клиентов</h2>
                <div class="row">
                  <div class="col-md-3">
                    <div><img src="img/10.png" alt=""></div>
                    <p>Дополнительная скидка 10% <br>
                        на вторую запчасть <br>
                        и любые последующие
                        </p>
                  </div>
                  <div class="col-md-3">
                    <div><img src="img/20.png" alt=""></div>
                    <p>Скидка 20% при заключении <br>
                        договора на годовое <br>
                        сервисное обслуживание
                        
                        </p>
                  </div>
                  <div class="col-md-3">
                    <div><img src="img/delivery-truck (4).png" alt=""></div>
                    <p>Бесплатная доставка <br>
                        по всей России<br>
                        для всех клиентов
                        </p>
                  </div>
                  <div class="col-md-3">
                    <div><img src="img/telemarketer.png" alt=""></div>
                    <p>Бесплатная круглосуточная <br>
                        техническая поддержка
                        </p>
                  </div>
                </div>
          </div>
        </section>
<!-- ===================================================================== -->
<section class="questions block_padding">
  <div class="container">
    <h2>У Вас остались вопросы?</h2>
    <h3>На Ваши вопросы отвечает Эдуард</h3>
   
    <div class="faq">
      <div class="item">
          <div class="ask">1. Как заказать деталь правильно?</div>
          <div class="answer">
              <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ab neque optio velit amet dignissimos recusandae voluptatum earum nemo odio sit, ipsa consequatur cumque eum numquam animi, excepturi nesciunt dicta hic repellat atque eos sapiente! Illum qui</p>
          </div>
      </div>
      <div class="item">
          <div class="ask">2. Что делать, если запчасть не понадобилась?</div>
          <div class="answer">
              <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ab neque optio velit amet dignissimos recusandae voluptatum earum nemo odio sit, ipsa consequatur cumque eum numquam animi, excepturi nesciunt dicta hic repellat atque eos sapiente! Illum qui</p>
          </div>
      </div>
      <div class="item">
          <div class="ask">3. В чем преимущества оригинальных деталей?</div>
          <div class="answer">
              <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ab neque optio velit amet dignissimos recusandae voluptatum earum nemo odio sit, ipsa consequatur cumque eum numquam animi, excepturi nesciunt dicta hic repellat atque eos sapiente! Illum qui</p>
          </div>
      </div>
      <div class="item">
          <div class="ask">4. Как правильно оформить заказ?</div>
          <div class="answer">
              <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ab neque optio velit amet dignissimos recusandae voluptatum earum nemo odio sit, ipsa consequatur cumque eum numquam animi, excepturi nesciunt dicta hic repellat atque eos sapiente! Illum qui</p>
          </div>
      </div>
      <div class="item">
          <div class="ask">5. В чем преимущества оригинальных деталей?</div>
          <div class="answer">
              <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Ab neque optio velit amet dignissimos recusandae voluptatum earum nemo odio sit, ipsa consequatur cumque eum numquam animi, excepturi nesciunt dicta hic repellat atque eos sapiente! Illum qui</p>
          </div>
      </div>
  </div>
  </div>
</section>

<!-- ===================================================================== -->
        <section class="work_with_us block_padding">
          <div class="container">
          <h2>С нами удобно работать!</h2>
          <p>Более 70% клиентов приходят к нам по рекомендации!</p>
          <div class="row work_with_us_block">
            <div class="col-sm-4">
              <div><img src="img/cash_card.png" alt=""></div>
              <p>Вы вносите предоплату <br>
                  в размере 50% <br>
                  от стоимости запчасти</p>
            </div>
            <div class="col-sm-4">
              <div><img src="img/truck_time.png" alt=""></div>
              <p>Отправляем запчасти <br>
                  в день заказа <br>
                  </p>
            </div>
            <div class="col-sm-4">
              <div><img src="img/list.png" alt=""></div>
              <p>Получаете деталь, <br>
                  проверяете, оплачиваете <br>
                  оставшиеся 50%</p>
            </div>
          </div>
          </div>
        </section>
<!-- ============================================================================== -->
      <section class="sklad block_padding">
        <div class="container">
          <h2>У нас собственный склад!</h2>
          <p>Это позволяет нам быстро обрабатывать запросы и отгружать детали <br>
              в день заказа!
              </p>

      <div id="carouselExampleFade" class="carousel slide carousel-fade" data-ride="carousel">
          <div class="carousel-inner">
            <div class="carousel-item active">
              <img src="img/slider1.jpg" class="d-block w-100" alt="...">
            </div>
            <div class="carousel-item">
              <img src="img/slider2.jpg" class="d-block w-100" alt="...">
            </div>
            <div class="carousel-item">
              <img src="img/slider3.jpg" class="d-block w-100" alt="...">
            </div>
          </div>
          <a class="carousel-control-prev" href="#carouselExampleFade" role="button" data-slide="prev">
            <span class="carousel-control-prev-icon" aria-hidden="true"></span>
            <span class="sr-only">Previous</span>
          </a>
          <a class="carousel-control-next" href="#carouselExampleFade" role="button" data-slide="next">
            <span class="carousel-control-next-icon" aria-hidden="true"></span>
            <span class="sr-only">Next</span>
          </a>
        </div>
        </div>
      </section>
<!--============================================================================ -->
<section class="service block_padding">
  <div class="container">
    <h2>У нас в штате есть мобильные бригады, <br>
        готовые выехать на сервисное обслуживание, <br>
        ремонт и проведение диагностики!</h2>
    <div class="row">
      <div class="col-lg-4">
        <div><img src="img/circle_truck.png" alt=""></div>
        <p>Организуем выезд специалистов <br>
            в течение двух дней</p>
      </div>
      <div class="col-lg-4">
        <div><img src="img/circle_bag.png" alt=""></div>
        <p>Бригады укомплектованы <br>
            всем необходимым оборудованием <br>
            для производства работ <br>
            любой сложности</p>
      </div>
      <div class="col-lg-4">
        <div><img src="img/certificate.png" alt=""></div>
        <p>Все специалисты проходят обучение <br>
            и ежегодную аттестацию</p>
      </div>
    </div>
  </div>
</section>
<!-- ================================================================================== -->
<section class="consultation block_padding">
  <div class="container">
    <h3>Если Вам необходима бесплатная консультация по вопросу обслуживания</h3>
    <p>Звоните нам по телефону:</p>
    <h2>+7 (818) 247-47-25</h2>
    <div class="consultation_form">
      <input type="text" placeholder="Введите ваш телефон">
      <button>Перезвонить Сейчас</button>
    </div>
  </div>
</section>
<!-- ======================================================================================= -->
<section class="office block_padding">
  <div class="container">
    <h3>Приезжайте в наш офис!</h3>
    <p>г. Архангельск, Москва-Сити, оф. 3725</p>
    <h3>+7 (818) 247-47-25</h3>
  </div>
</section>
<!-- ====================================================================================== -->
<section class="location">
<img src="img/location.jpg" alt="">
</section>
<!-- ====================================================================================== -->
<footer>
  <div class="container">
    <div class="row">
      <div class="col-lg-4">
        <img src="img/logo.png" alt="">
      </div>
      <div class="col-lg-4">
        <p>© SERMAX</p>
      </div>
      <div class="col-lg-4">
        <h5>Есть вопросы? Звоните!</h5>
        <h3>+7 (818) 247-47-25</h3>
      </div>
    </div>
  </div>
</footer>


    <script src="bootstrap-4/js/jquery_bootstrap.js"></script>
    <script src="bootstrap-4/js/popper_min.js"> </script>
    <script src="bootstrap-4/js/bootstrap_min.js"></script>
</body>
</html>
