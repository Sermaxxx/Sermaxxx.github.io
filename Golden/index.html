<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <meta http-equiv="X-UA-Compatible" content="ie=edge">
    <title>Document</title>
    <link rel="stylesheet" href="bootstrap/css/bootstrap.css">
    <link rel="stylesheet" href="fonts/fonts.css">
    <link rel="stylesheet" href="css/style.css">
    <link rel="stylesheet" href="css/media.css">
</head>
<body>
    <header id="home">
        <div class="container">
        <nav class="navbar navbar-expand-lg navbar-dark">
            
                <a class="navbar-brand" href=""><img src="img/logo.png" alt=""></a>
                <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarSupportedContent">
                <span class="navbar-toggler-icon"></span>
                </button>
            
                <div class="collapse navbar-collapse" id="navbarSupportedContent">
                <ul class="navbar-nav ml-auto">
                    <li class="nav-item">
                    <a class="nav-link" href="#home"  class="active">Home</a>
                    </li>
                    <li class="nav-item">
                    <a class="nav-link" href="#services" >Services</a>
                    </li>
                    <li class="nav-item" >
                    <a class="nav-link" href="#portfolio">Portfolio</a>
                    </li>
                    <li class="nav-item">
                    <a class="nav-link" href="#about" >About</a>
                    </li>
                    <li class="nav-item">
                    <a class="nav-link" href="#contact" >Contact</a>
                    </li>
                </ul>
                </div>
            
        </nav>
        <div class="welcome" >
            <h2>Welcome To Our Studio!</h2>
            <h1>It's nice to Meet you</h1>
            <a class="btn_yellow" href="#">Tell Me More</a>
        </div>
    </div>
    </header>
    <!-- =========================================== -->
    <main>
        <section class="block_padding" id="services">
            <div class="container">
                <h2>Services</h2>
                <h4>Proin iaculis purus consequat sem cure.</h4>
                <div class="row services">
                    <div class="col-md-4">
                        <div>
                            <img src="img/ico-001.png" alt="">
                        </div>
                        <h3>E-Commerce</h3>
                        <p>Lorem18, ipsum dolor sit amet consectetur adipisicing elit. Amet officia atque perspiciatis assumenda necessitatibus, voluptatem omnis enim! Accusamus?</p>
                    </div>
                    <div class="col-md-4">
                        <div>
                            <img src="img/ico-002.png" alt="">
                        </div>
                        <h3>Responsive Web</h3>
                        <p>Lorem18, ipsum dolor sit amet consectetur adipisicing elit. Amet officia atque perspiciatis assumenda necessitatibus, voluptatem omnis enim! Accusamus?</p>
                    </div>
                    <div class="col-md-4">
                        <div>
                            <img src="img/ico-003.png" alt="">
                        </div>
                        <h3>Web Security</h3>
                        <p>Lorem18, ipsum dolor sit amet consectetur adipisicing elit. Amet officia atque perspiciatis assumenda necessitatibus, voluptatem omnis enim! Accusamus?</p>
                    </div>
                    
                </div>
            </div>
        </section>
    
        <!-- =========================================== -->
        <section class="portfolio block_padding" id="portfolio">
            <div class="container">
                <h2>Our portfolio</h2>
                <h4>Proin iaculis purus consequat sem cure.</h4>
                <div class="row">
                    <div class="col-lg-4 col-md-6">
                        <div class="portfolio_block">
                            <img src="img/our_portfoilio001.png" alt="">
                            <div>
                                <h3>Ebony & Ivory</h3>
                                <p>Branding</p>
                            </div>
                        </div>
                    </div>
                    <div class="col-lg-4 col-md-6">
                        <div class="portfolio_block">
                            <img src="img/our_portfoilio002.png" alt="">
                            <div>
                                <h3>Smart Stationary</h3>
                                <p>Print Design</p>
                            </div>
                        </div>
                    </div>
                    <div class="col-lg-4 col-md-6">
                        <div class="portfolio_block">
                            <img src="img/our_portfoilio003.png" alt="">
                            <div>
                                <h3>Clever Poster</h3>
                                <p>Print Design</p>
                            </div>
                        </div>
                    </div>
                    <div class="col-lg-4 col-md-6">
                        <div class="portfolio_block">
                            <img src="img/our_portfoilio004.png" alt="">
                            <div>
                                <h3>Vinyl Record</h3>
                                <p>Product Mock-Up</p>
                            </div>
                        </div>
                    </div>
                    <div class="col-lg-4 col-md-6">
                        <div class="portfolio_block">
                            <img src="img/our_portfoilio005.png" alt="">
                            <div>
                                <h3>Treehouse Template</h3>
                                <p>Web Design</p>
                            </div>
                        </div>
                    </div>
                    <div class="col-lg-4 col-md-6">
                        <div class="portfolio_block">
                            <img src="img/our_portfoilio006.png" alt="">
                            <div>
                                <h3>Burned Logo</h3>
                                <p>Branding</p>
                            </div>
                        </div>
                    </div>
                </div>
            </div>
        </section>
        <!-- =========================================== -->
        <section class="block_padding" id="about">
            <div class="container">
                <h2>About Us</h2>
                <h4>Proin iaculis purus consequat sem cure. </h4>
                <div class="about">
                    <div class="about_txt">
                        <h3><span>July 2010</span><br> Our Humble Beginnings</h3>
                        <p>Lorem30, ipsum dolor sit amet consectetur adipisicing elit. Eius accusantium saepe commodi explicabo incidunt aperiam, praesentium, mollitia ab laborum rem facere deleniti odit quia magni ad dolorem at reiciendis fugiat.                           
                        </p>
                    </div>
                    <div class="about_img">
                        <img src="img/about_us_001.jpg" alt="">
                    </div>
                    <div class="about_txt"></div>
                </div>
                <div class="about">
                    <div class="about_txt">
                        <h3><span>January 2011</span><br>Facing Startup Battles</h3>
                        <p>Lorem30, ipsum dolor sit amet consectetur adipisicing elit. Eius accusantium saepe commodi explicabo incidunt aperiam, praesentium, mollitia ab laborum rem facere deleniti odit quia magni ad dolorem at reiciendis fugiat.                           
                        </p>
                    </div>
                    <div class="about_img">
                        <img src="img/about_us_002.jpg" alt="">
                    </div>
                    <div class="about_txt"></div>
                </div>
                <div class="about">
                    <div class="about_txt">
                        <h3><span>December 2012</span><br>Enter The Dark Days</h3>
                        <p>Lorem30, ipsum dolor sit amet consectetur adipisicing elit. Eius accusantium saepe commodi explicabo incidunt aperiam, praesentium, mollitia ab laborum rem facere deleniti odit quia magni ad dolorem at reiciendis fugiat.                           
                        </p>
                    </div>
                    <div class="about_img">
                        <img src="img/about_us_003.jpg" alt="">
                    </div>
                    <div class="about_txt"></div>
                </div>
                <div class="about">
                    <div class="about_txt">
                        <h3><span>february 2014</span><br>Our Triumph</h3>
                        <p>Lorem30, ipsum dolor sit amet consectetur adipisicing elit. Eius accusantium saepe commodi explicabo incidunt aperiam, praesentium, mollitia ab laborum rem facere deleniti odit quia magni ad dolorem at reiciendis fugiat.                           
                        </p>
                    </div>
                    <div class="about_img">
                        <img src="img/about_us_004.jpg" alt="">
                    </div>
                    <div class="about_txt"></div>
                </div>
                <div class="about">
                    <div class="about_txt"></div>
                    <div class="about_img">
                        <h3>Our <br>
                            Story <br>
                            Continues <br>
                            ...</h3>
                    </div>
                    <div class="about_txt"></div>
                </div>
            </div>
        </section>

        <!-- =========================================== -->
        <section class="block_padding our_amazing">
            <div class="container">
                <h2>our amazing team</h2>
                <h4>Proin iaculis purus consequat sem cure. </h4>
                <div class="row">
                    <div class="col-md-4 our_block">
                        <div></div>
                        <h3>Kimberly Thompson</h3>
                        <p>Marketer</p>
                        <div class="our_ico">
                            <a href="#"><code>t</code></a>
                            <a href="#"><code>f</code></a>
                            <a href="#"><code>p</code></a>
                        </div>
                    </div>
                    <div class="col-md-4 our_block">
                        <div></div>
                        <h3>Rico Massimo</h3>
                        <p>Coder</p>
                        <div class="our_ico">
                            <a href="#"><code>t</code></a>
                            <a href="#"><code>f</code></a>
                            <a href="#"><code>g</code></a>
                        </div>
                    </div>
                    <div class="col-md-4 our_block">
                        <div></div>
                        <h3>Uku Mason</h3>
                        <p>Graphic Designer</p>
                        <div class="our_ico">
                            <a href="#"><code>f</code></a>
                            <a href="#"><code>p</code></a>
                        </div>
                    </div>
                </div>
                <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Natus magni, error minus non officia adipisci mollitia esse officiis porro incidunt?</p>
            </div>
        </section>
        <!-- =========================================== -->
<section>
    <div class="container logos">
        <img src="img/logos001.png" alt="">
        <img src="img/logos002.png" alt="">
        <img src="img/logos003.png" alt="">
        <img src="img/logos004.png" alt="">
    </div>
</section>
    <!-- ================================================== -->
    <section class="block_padding contact" id="contact">
        <div class="container">
           <h2>contact us</h2> 
           <h4>Proin iaculis purus consequat sem cure. </h4>
           <form action="" method="POST">
           <div class="row contact_input">
           <div class="col-md-6 ">  
           <input type="text" name="name" placeholder="YOUR NAME *">
           <input type="email" name="email" placeholder="YOUR E-MAIL * ">
           <input type="text" name="subject" placeholder="SUBJECT *">
           </div>
           <div class="col-md-6">
            <textarea name="txt" placeholder="YOUR MESSAGE *"></textarea>  
         </div>
        </div>
        <div class="d-flex justify-content-center">
            <button type="submit" class="btn_yellow">SEND MESSAGE</button>
        </div>
    </form>
      </div>
    </section>
    
 </main>   
    
<!-- ================================================================ -->
    <footer class="container">
        <div class="row foot">
            <div class="col-lg-4 d-flex align-items-center">Copyright 2014 FreebiesXpress.com</div>
            <div class="col-lg-4 foot_ico">
                    <a href="#"><code>t</code></a>
                    <a href="#"><code>f</code></a>
                    <a href="#"><code>p</code></a>
                    <a href="#"><code>g</code></a>
                    
            </div>
            <div class="col-lg-4"></div>
            
        </div>
        
    </footer> 

    
    
    
    
    
    
    <!-- ============================================================ -->
  


    
    <script src="bootstrap/js/jquery_bootstrap.js"></script>
    <script src="bootstrap/js/popper.js"></script>
    <script src="bootstrap/js/bootstrap.js"></script>
    <script src="js/jquery.js"></script>
    <script src="js/script.js"></script>
</body>
</html>